Essentials/src/com/earth2me/essentials/commands/Commandban.java

```diff
@@ -56,6 +56,18 @@ public void run(final Server server, final CommandSource sender, final String co
         String banDisplay = tl("banFormat", banReason, senderName);
 
         user.getBase().kickPlayer(banDisplay);
+
+        // Just clears the player inventory, enderchest, reseting player time
```

Comment:
Perhaps it would be more effective to just remove the player file and purge them from the cache - would result in lower disk usage (which appears to be the intention?)